src/api/channels.test.tsx

```diff
@@ -314,7 +234,7 @@ describe('channels api functions', () => {
 
       expect(result2.current.isFetching).toBeFalsy();
       expect(result2.current.isLoading).toBeTruthy();
-      expect(axios.get).not.toHaveBeenCalled();
+      expect(requestSent).toBe(false);
     });
 
     it.todo(
```

Comment:
Is this test obsolete, or is it still pending implementation in `msw`?

Reply:
They're still pending the fact that we haven't decided how we want to handle errors from the API in OG yet - you can see we're currently just logging errors right now - so keeping these tests as `todo`s reminds us that there's no proper API error handling yet